pcbdraw/pcbdraw.py

```diff
@@ -867,6 +949,24 @@ def main():
         print(e)
         sys.exit(1)
 
+    # If some keys aren't in the loaded style compared to the default style, copy it from the default style
+    for s in default_style:
+        if s not in style:
+            style[s] = default_style[s]
+
+    tht_resistor_settings = {}
+    if args.resistor_values:
+        split_list = args.resistor_values.split(",")
+        for r in split_list:
+            r_s = r.split(":")
+            tht_resistor_settings[r_s[0]] = {'override_val': r_s[1]}
+    if args.resistor_flip:
+        for r in args.resistor_flip.split(","):
+            if r in tht_resistor_settings:
+                tht_resistor_settings[r]['flip'] = True
+            else:
+                tht_resistor_settings[r] = {'flip': True}
```

Comment:
Please consider factoring out this into a separate function.

Reply:
Same as above